linux_pipeline/Jenkinsfile_ubuntu_azure_kernel_validation

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/env groovy
+
+def PowerShellWrapper(psCmd) {
+    psCmd = psCmd.replaceAll("\r", "").replaceAll("\n", "")
+    def exitCode = bat (returnStatus: true, script: "powershell.exe -NonInteractive -ExecutionPolicy Bypass -Command \"\$ErrorActionPreference='Stop';[Console]::OutputEncoding=[System.Text.Encoding]::UTF8;$psCmd;EXIT \$global:LastExitCode\"")
+    return exitCode
+}
+
+properties ([
+    parameters([
+        string (name: 'Distro', defaultValue: '', 
+                description: 'xenial - only Xenial validation \n bionic - only Bionic validation \n both - both distros validation \n'),
+        string (name: 'Kernel', defaultValue: 'linux-azure',
+                description: 'linux-azure - only linux-azure kernel validation \n linux-azure-edge - only linux-azure-edge kernel validation \n both - both kernels validation \n'),
+        string (name: 'Platform', defaultValue: 'both',
+                description: 'hyperv - only Hyper-V validation \n azure - only Azure validation \n both - both platforms validation \n')
+    ]),
+    pipelineTriggers([cron('H */12 * * *')])
+])
+
+def distroToTest = []
+
+if ("${env:distro}" == "") {
+    stage ("Check for kernel update") {
+        node ("ws2016"){
+            withCredentials(bindings: [string(credentialsId: 'AZURE_KERNEL_INFO_LOCATION', variable: 'AZURE_KERNEL_INFO_LOCATION')]) {
+                dir (env.BUILD_NUMBER + env.BRANCH_NAME) { 
+                    checkout scm
+                    echo 'Checking if there is a new kernel version'
+                    kernel_sts = PowerShellWrapper('''
+                        & ".\\scripts\\ubuntu_azure_kernel\\get_ubuntu_azure_kernel_version.ps1"
+                            -RemoteLocation "${env:AZURE_KERNEL_INFO_LOCATION}"
+                    ''') // LatestVersionsFile changed
+                    if ("${kernel_sts}" == "1") {
+                        echo "New kernel available for Bionic and Xenial. Triggering testing for both"
+                        distroToTest.add('bionic')
```

Comment:
put 'bionic' and the likes to a constant to be easily changeable
DISTRO_BIONIC='bionic' etc